google-cloud-trace/lib/google/cloud/trace/v1/doc/google/devtools/cloudtrace/v1/trace.rb

```diff
@@ -0,0 +1,187 @@
+        #     collected from the application.
+        # @!attribute [rw] filter
+        #   @return [String]
+        #     An optional filter for the request.
+        # @!attribute [rw] order_by
+        #   @return [String]
+        #     Field used to sort the returned traces. Optional.
+        #     Can be one of the following:
+        #
+        #     *   +trace_id+
+        #     *   +name+ (+name+ field of root span in the trace)
+        #     *   +duration+ (difference between +end_time+ and +start_time+ fields of
+        #          the root span)
+        #     *   +start+ (+start_time+ field of the root span)
+        #
+        #     Descending order can be specified by appending +desc+ to the sort field
+        #     (for example, +name desc+).
+        #
+        #     Only one sort field is permitted.
+        class ListTracesRequest
+          # Type of data returned for traces in the list.
+          module ViewType
+            # Default is +MINIMAL+ if unspecified.
+            VIEW_TYPE_UNSPECIFIED = 0
+
+            # Minimal view of the trace record that contains only the project
+            # and trace IDs.
+            MINIMAL = 1
+
+            # Root span view of the trace record that returns the root spans along
+            # with the minimal trace data.
+            ROOTSPAN = 2
+
+            # Complete view of the trace record that contains the actual trace data.
+            # This is equivalent to calling the REST +get+ or RPC +GetTrace+ method
+            # using the ID of each listed trace.
+            COMPLETE = 3
+          end
+        end
+
+        # The response message for the +ListTraces+ method.
+        # @!attribute [rw] traces
+        #   @return [Array<Google::Devtools::Cloudtrace::V1::Trace>]
+        #     List of trace records returned.
+        # @!attribute [rw] next_page_token
+        #   @return [String]
+        #     If defined, indicates that there are more traces that match the request
+        #     and that this value should be passed to the next request to continue
+        #     retrieving additional traces.
+        class ListTracesResponse; end
+
+        # The request message for the +GetTrace+ method.
+        # @!attribute [rw] project_id
+        #   @return [String]
+        #     ID of the Cloud project where the trace data is stored.
+        # @!attribute [rw] trace_id
+        #   @return [String]
+        #     ID of the trace to return.
+        class GetTraceRequest; end
+
+        # The request message for the +PatchTraces+ method.
+        # @!attribute [rw] project_id
+        #   @return [String]
+        #     ID of the Cloud project where the trace data is stored.
+        # @!attribute [rw] traces
+        #   @return [Google::Devtools::Cloudtrace::V1::Traces]
+        #     The body of the message.
+        class PatchTracesRequest; end
+      end
+    end
+  end
+end
```

Comment:
@swcloud I think it would be best to add `.rubocop.yml` at this time (to this PR), although it can contain exclusions for the current GAPIC code, since #1078 shows that when library code is later added, `.rubocop.yml` might be forgotten.